gbus/builder/builder.go

```diff
@@ -107,6 +107,7 @@ func (builder *defaultBuilder) Build(svcName string) gbus.Bus {
 			panic(err)
 		}
 	}
+
```

Comment:
file contains no changes (other than a blank line) should be reverted and not included in the PR